src/Command/GetPickupPointsByCarriersAndCountryCodes.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MwlBundle\Command;
+
+use Answear\MwlBundle\Client\Client;
+use Answear\MwlBundle\Request\GetPickupPointsByCarriersAndCountryCodesRequest;
+use Answear\MwlBundle\Response\GetPickupPointsResponse;
+
+readonly class GetPickupPointsByCarriersAndCountryCodes extends AbstractCommand
+{
+    public function __construct(private Client $client)
```

Comment:
What about change AbstractCommand as non Abstract, and move this constructor there? All three commands will have only Request method.

Reply:
good idea :+1: though AbstractCommand can remain abstract